### PR TITLE
Remove redundand overrides causing confusing DEBUG logging (#bsc1189501)

### PR DESCRIPTION
**Pull request by ycedres**
### What does this PR do?

Removes redundant overrides of ip_address and ip_interface that are causing confusing DEBUG logging when IPv6 interfaces are present.

### What issues does this PR fix or reference?
Fixes: [#1189501](https://bugzilla.suse.com/show_bug.cgi?id=1189501)
https://github.com/SUSE/spacewalk/issues/17484

### Commits signed with GPG?
Yes

### CODE REVIEW

**Review by agraul — APPROVED** [review]
Great first PR, it looks very good! :tada:

**Comment by agraul** [comment]
@ycedres is this also needed for other Salt versions, e.g. 3000 (for Python 2 based Salt) or 3000.3 (for Python 2 based Salt SSH)?

**Review by vzhestkov — APPROVED** [review]
Thanks @ycedres :+1:

File: salt/_compat.py [hunk 1]
```diff
@@ -242,88 +242,7 @@ def __init__(self, address):
...
-        log.debug("Error while getting IPv6 interface for address %s", address)
-        log.debug(err)
-
-    raise ValueError(
-        "{} does not appear to be an IPv4 or IPv6 interface".format(address)
-    )
-
-
 if ipaddress:
     ipaddress.IPv6Address = IPv6AddressScoped
```
[the remark this answers is not in the document]
Comment (ycedres) [reply salt/_compat.py:246]: Thank you Victor. I did check and the behaviour of IPv6Address in the Python module is different as an scope property is added in IPv6AddressScoped.

**Pull request merged**